src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py

```diff
@@ -1871,178 +1752,6 @@ def setUp(self):
         conf.allowed_years = [2020]
         conf.save()
 
-    def test_import_zaaktype_already_exists(self, *mocks):
```

Comment:
I removed this because I think it's no longer relevant after the change to always preserve UUIDs

Reply:
re-added them, because it's now optional to generate new UUIDs